File: sdkwork-app-sdk-typescript/src/types/plus-api-result-experiment-report-vo.ts
import type { ExperimentReportVO } from './experiment-report-vo';

/** API调用结果 */
export interface PlusApiResultExperimentReportVO {
  /** Response data */
  data: ExperimentReportVO;
  /** Response code: 2000=success, 4xxx=business failure, 5xxx=server error */
  code: string;
  /** Business message */
  msg: string;
  /** Request identifier */
  requestId: string;
  /** Client IP address */
  ip?: string;
  /** Server hostname */
  hostname?: string;
  /** Business error name */
  errorName: string;
}

File: sdkwork-app-sdk-python/sdkwork_app_sdk/models/plus_api_result_experiment_report_vo.py
from dataclasses import dataclass
from typing import Optional, List, Dict, Any

@dataclass
class PlusApiResultExperimentReportVO:
    """API调用结果"""
    data: ExperimentReportVO
    code: str
    msg: str
    request_id: str
    ip: str = None
    hostname: str = None
    error_name: str
